FirebaseAuth/Tests/Unit/AuthBackendRPCImplentationTests.swift

```diff
@@ -590,12 +590,11 @@ class AuthBackendRPCImplementationTests: RPCBaseTests {
         try self.rpcIssuer.respond(withJSON: [:])
       }
       _ = try? await rpcImplementation.call(with: request)
-      // Make sure completeRequest updates.
-      usleep(10000)
```

Comment:
I beliee this leads to flakes on slower CI machines where this is not a long enough wait.